connect/transforms/src/main/java/org/apache/kafka/connect/transforms/SetSchemaMetadata.java

```diff
@@ -154,4 +161,14 @@ protected static Object updateSchemaIn(Object keyOrValue, Schema updatedSchema)
         }
         return keyOrValue;
     }
+
+    private static String buildSchemaName(String namespace, String name) {
```

Comment:
I know that we're trying to cover all of the cases in the tests that call the transform, but it'd be good to also have test case methods that check the different permutations of inputs to this method, including null, empty strings, and blank strings for names and namespaces, namespaces with and without a trailing `.`, namespaces with multiple `.` characters, namespaces with leading and/or trailing whitespaces, etc. 

In addition to such tests being easier to write and understand, they help test all possible combinations or code paths _and_ they help identify expected behaviors in various scenarios to better catch regressions in the future.